blocks/resizer-image-block/extractImageFromStory.js

```diff
@@ -0,0 +1,35 @@
+const extractImageFromPromo = (promoItem) => (
+  (promoItem?.basic?.type === 'image' && promoItem?.basic?.url)
+    ? promoItem.basic.url
+    : null
+);
+
+/**
+ * Helper to resolve an image from an story.
+ *
+ * @param storyObject ANS story
+ *
+ * @return an string with the image URL or null if not found
+ */
+const extractImageFromStory = (storyObject) => {
+  const { promo_items: promoItems } = storyObject;
```

Comment:
We can probably deconstruct the `lead_art` properties of `promoItems` here

Reply:
I tried, but is a bit more tricky because of the 2nd helper `extractImageFromPromo` (to keep promo_items and lead_art deconstructed)